pkg/kv/kvserver/store.go

```diff
@@ -1533,6 +1533,18 @@ func NewStore(
 		s.consistencyLimiter.UpdateLimit(quotapool.Limit(rate), rate*consistencyCheckRateBurstFactor)
 	})
 
+	useDefaultConnectionClassForRaft.SetOnChange(&cfg.Settings.SV, func(ctx context.Context) {
+		connClass := defRaftConnClass(&cfg.Settings.SV)
+		log.Infof(ctx,
+			"updating raft transport connection class on all replicas to : %v", connClass,
+		)
+		s.VisitReplicas(func(r *Replica) (wantMore bool) {
+			r.connectionClass.set(connClass)
```

Comment:
Need to use the same logic as in `setDescLockedRaftMuLocked`, because the `ConnectionClassForKey` distinguishes the "system" class. Also, if the replica is uninitialized, we shouldn't touch it (it's at zero value which is `DefaultClass`).